site/src/component/GradeDist/GradeDist.tsx

```diff
@@ -75,7 +75,8 @@ const GradeDist: FC<GradeDistProps> = (props) => {
    */
   const createProfEntries = useCallback(() => {
     const professors: Set<string> = new Set();
-    const result: Entry[] = [];
+    const allInstructors = { value: 'All Instructors', text: 'All Instructors' };
+    const result: Entry[] = [allInstructors];
```

Comment:
```suggestion
    const result: Entry[] = [{ value: 'All Instructors', text: 'All Instructors' }];
```

You can fold allInstructors while initializing result to match how ``createQuarterEntries`` does it below.